bin/gwdetchar-scattering

```diff
@@ -472,8 +475,8 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
         page.tr.close()
         page.tr()
         page.th('Efficiency<br><small>(SNR&ge;8 and '
-                'f<sub>peak</sub>&lt;%.2f Hz)</small>'
-                % (2 * args.frequency_threshold))
+                '%.2f Hz</sub>&ltf<sub>peak</sub>&lt;%.2f Hz)</small>'
+                % (10*1,4 * args.frequency_threshold))
```

Comment:
@siddharth101, this can be cleaned up as well:

```suggestion
                % (10, args.frequency_threshold * 4))
```

Reply:
Yup, will do now.

Reply:
Thank you, and sorry for the amount of comments, I think I'll have at most one or two more (I'm currently looking into the best way to simplify the logic towards the bottom)

Reply:
This has been really helpful, and i apologize for so many commits for a small change. I will be able to do much better as I get a hang of it.

Reply:
That's ok! I'm hoping you're getting something out of it, you did a good job of identifying what needed to be changed and putting something together that works. The segment logic is also a really interesting corner case 😀